### DISCUSSION TITLE
OPINION POLL.

### DISCUSSION THREAD

**Discussion by Optimus523** [Cyfrin/foundry-full-course-cu#3103] [General]
Okay, I just got done with Blockchain basics and my next destination is supposed to be learning solidity. So here is the big deal elder brother opted to learn solidity meanwhile I instead learn python and vyper. I see it as a good idea but don't know if it's the right way, just needed some advice.

**Comment by EngrPips** [comment]
Honestly, whichever one you learn is still a good choice, and at the end of the day, it is only syntax that majorly differs. The logic and principle of programming remain the same. So if you are excellently good at one, you can easily learn any other langugae.

**Reply by Optimus523** [reply]
Alright then, I appreciate your input.

**Reply by EngrPips** [reply]
Anytime.
